032-MLOpsFromScratch/README.md

```diff
@@ -0,0 +1,116 @@
+
+- What is time series data and time series forecasting?
+
+  - How is that different from conventional ML modeling?
+
+  - How is a forecasting task different from other tasks?
+
+- How to enable Continuous Integration for our AI project by creating a Build
+  pipeline? What artifacts do we deploy into the repo and what is our
+  architecture end state?
+
+- How to enable Continuous Delivery for our AI project by creating a Release
+  pipeline? How do we automate our ML lifecycle and score our data?
+
+- What is retraining? How to evaluate the best model between the current model
+  and previous models?
+
+- How to convert a model into an endpoint?
+
+- How to identify Data Drift?
+
+Your team will have individuals of varying skill sets, so it is imperative to decide who will take on the following roles:
+(1) Data Scientist
+(2) ML Engineer
+(3) DevOps Engineer
+(4) Data/Business Analyst
+
+## Challenges
+
+1.  [Challenge 1: Incorporate your locally trained machine learning code into Azure DevOps](Student/01-TimeSeriesForecasting.md)
+
+2.  [Challenge 2: Create a Unit Test in Azure
+    DevOps](Student/02-UnitTesting.md)
+
+3.  [Challenge 3: Create a Build pipeline in Azure
+    DevOps](Student/03-BuildPipeline.md)
+
+4.  [Challenge 4: Create a Release pipeline in Azure
+    DevOps](Student/04-ReleasePipeline.md)
+
+5.  [Challenge 5: Retraining and Model
+    Evaluation](Student/05-RetrainingAndEvaluation.md)
+
+## Prerequisites
+
+1.  Azure subscription. If you do not have one, you can sign up for a [free trial](https://azure.microsoft.com/en-us/free/).  
+
+2.  [Azure Machine Learning service workspace](https://ml.azure.com/) - It is a foundational resource in
+    the cloud that you use to experiment, train, and deploy machine learning
+    models.
+
+3.  Azure DevOps subscription. If you do not have one, you can sign up for a
+    [free account](https://azure.microsoft.com/en-us/services/devops/).
+
+    - Install [Azure DevOps Machine Learning
+      extension](https://marketplace.visualstudio.com/items?itemName=ms-air-aiagility.vss-services-azureml)
+
+    - Request Admin access to [create Service Connections](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) within Azure DevOps to connect with Azure ML Workspace. If the access is not granted, have admin create those service connections ahead of time. To do that, create new service connections in Project Settings for your Azure ML service and Azure Subscription using service principal. 
+    Note: Azure ML Workspace must be created first before creating Service Connections. 
+  
+4.  Python Installation, version at least \>= 3.6.5. Anaconda is more preferred
+    for Data Science tasks.
+
+    - Anaconda - <https://docs.anaconda.com/anaconda/install/windows/>
+
+    - Miniconda - <https://docs.conda.io/en/latest/miniconda.html>
+
+    - <https://www.python.org/downloads/release/python-3611/>

```

Comment:
Yes, this is one thing we would like to leave it as is -- we like to standardize the version to 3.6.5 only to avoid any unforeseen issues with the latest versions.